overrides.nix

```diff
@@ -20,11 +20,18 @@ in {
 
   Glob = whenGhcjs dontCheck super.Glob;
 
-  aeson = dontCheck (callHackageDirect {
-    pkg = "aeson";
-    ver = "1.4.5.0";
-    sha256 = "0imcy5kkgrdrdv7zkhkjvwpdp4sms5jba708xsap1vl9c2s63n5a";
-  });
+  aeson = if self.ghc.isGhcjs or false
+    then dontCheck (self.callCabal2nix "aeson" (pkgs.fetchFromGitHub {
+        owner = "obsidiansystems";
+        repo = "aeson";
+        rev = "d6288c431a477f9a6e93aa80454a9e1712127548"; # branch v1450-text-jsstring containing (ToJSVal Value) instance and other fixes
+        sha256 = "102hj9b42z1h9p634g9226nvs756djwadrkz9yrb15na671f2xf4";
+      }) {})
+    else dontCheck (callHackageDirect {
```

Comment:
We merged a change that adds the `cffi` cabal flag to aeson.  See

https://github.com/kadena-io/pact/commit/593938a41588cb35b36afc41eed66329ff166f0d#diff-c52519d307faa915611490bbbeb44308

That should be merged in here.

Reply:
I did the merge and fixed this.

Reply:
Great, thanks